zyte_test_websites/jobs/extraction.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+from web_poet import field
+from zyte_common_items import (
+    BaseSalary,
+    HiringOrganization,
+    JobLocation,
+    JobPostingNavigationPage,
+    JobPostingPage,
+    ProbabilityRequest,
+    Request,
+)
+
+if TYPE_CHECKING:
+    from parsel import Selector, SelectorList
+
+
+class TestJobPostingPage(JobPostingPage):
+    @field
+    def datePublished(self) -> str | None:
+        return datetime.strptime(self.datePublishedRaw, "%b %d, %Y").isoformat()
```

Comment:
Not about this PR, really, but it feels like we should have zyte-common-items automatically convert a datetime to a string like this.

Reply:
Actually, zyte-common-items has some related code but it's in `zyte_common_items._dateutils` (with a "backwards compatibility import" in `zyte_common_items.util` for one function) so I'm not sure if external code is supposed to use it. cc @kmike

Reply:
I think that's fine to use it, and we should make it more public.
+1 to Adrian about dding more out-of-box processing.